### ISSUE TITLE
Item Details accessibility issues

### ISSUE THREAD

**Issue by struthj** [SmarterApp/AP_ItemSampler#2]
- [x] The following changes should be made to the page's heading structure (Severity 3): 
No `<h1>` is present. Provide a descriptive first-level heading at the start of the main page area. 
Change "Sample Items" `<h3>` to `<h2>.` 
Change the item results `h4>` to` <h3>. `

- [x] The "Accessibility" button is presented to a screen reader as a link, but it should be presented as a button, just like the other buttons on this page that open dialogs. Severity 3 

EDIT: This appears on http://sampleitems.smarterbalanced.org, but is fixed/ not present on this current branch Acessibility modal appears as a button

- [x] This page includes a section with role="toolbar". This role should be used for a complex application toolbar that you navigate with arrow keys, not a group of buttons. It makes it much more difficult for a screen reader user to read through this section. Remove this role. Severity 2 

- [x] This page includes three groups with aria-label values of  "Toolbar with button groups" "First group" and "Second group". These labels increase overhead more than they help. They should be removed. Severity 4 

- [x] The "Share" dialog contains a text field with no label. The text that precedes the field "The following URL…" should be marked up as a <label> for this field (<label for="shareUrl">The following URl…<input id="shareUrl"…>). Severity 2 

- [x] The "Cancel" button in the "Accessibility" dialog has white text on a blue (#0085ad) background. This text has 4.2:1 contrast. WCAG 2.0 requires contrast of at least 4.5:1 for this element. Severity 3

EDIT changed color to #005670; contrast ratio is 8.19:1 well above 4.5:1

- [x] In the "Accessibility Options" dialog, light-gray (#999999) text is used for disabled text styled with the class="selection-disabled" and has 2.9:1 contrast. Disabled text is exempted from WCAG 2.0 contrast requirements because it must have low contrast so that it is not confused with areas that are not disabled. However, we recommend providing text with slightly more contrast if possible, so that it is more legible. Severity 4 

EDIT increased Contrast Ratio: 3.45:1 with #8a8a8a

**Comment by struthj** [comment]
Assuming that "Change the item results h4> to<h3>." means the Item Cards in Browse items
